backend/src/events/column-events.ts

```diff
@@ -14,10 +14,45 @@ import {
   UPDATE_BOARD,
   EDIT_COLUMN,
   BOARD_ERROR,
+  TOGGLE_COLUMN_BLUR,
 } from "./event-names";
+import { getBoard, saveBoard } from "../repositories";
+import { RetroBoard } from "../models/RetroBoard";
 
 const UTF8 = "utf8";
 
+function handleBoardError(client: Socket, eventName: string, err: Error) {
+  logError(TOGGLE_COLUMN_BLUR, err);
+  client.emit(BOARD_ERROR);
+}
+
+function blurColumn(board: RetroBoard, columnId: string) {
+  const column = board.columns[columnId];
+  column.isBlurred = !column.isBlurred;
+  column.itemIds.forEach((itemId) => {
+    board.items[itemId].isBlurred = column.isBlurred;
+  });
+}
+
+export function toggleColumnBlur(
+  io: Server,
+  client: Socket,
+  roomId: string
+): void {
+  client.on(TOGGLE_COLUMN_BLUR, (boardId: string, columnId: string) => {
+    getBoard(boardId)
+      .then((board: RetroBoard) => {
```

Comment:
This could be rewritten using `async/await`. Is there a specific reason you did not use it?

Reply:
I am not really sure if socket.io supports this. I looked it up and there seems to be an open critical [issue](https://github.com/socketio/socket.io/issues/3431), causing the server to expose the payload to all connected clients when using async/await. 
But if you are certain that this will be not a problem, I can change the syntax.

Reply:
This shouldn't be a problem since the `await` would happen outside of `emit`.

Reply:
Done.